homeassistant/components/signal_messenger/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Signal Messenger",
   "documentation": "https://www.home-assistant.io/integrations/signal_messenger",
   "codeowners": ["@bbernhard"],
-  "requirements": ["pysignalclirestapi==0.3.4"],
+  "requirements": ["pysignalclirestapi==0.3.17"],
```

Comment:
Since this bumps a dependency. The pull request needs a link to the changelog between revisions.

Reply:
There doesn't appear to be a maintained changelog for this package (apologies if I'm misunderstanding your request):
https://github.com/bbernhard/pysignalclirestapi/releases
https://pypi.org/project/pysignalclirestapi/#history

Reply:
You can link to a diff between the commit of the releases.

Reply:
I've now added that as a comment to the PR